docs/user-guide/admin-cli.md

```diff
@@ -20,68 +20,74 @@ of their usage.
 
 ### Help
 ```bash
-[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant -h
-usage: vmdkops_admin.py tenant [-h] {access,rm,create,ls,vm} ...
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant -h
+usage: vmdkops_admin.py tenant [-h] {create,vm,update,access,ls,rm} ...
 
 positional arguments:
-  {access,rm,create,ls,vm}
-    access              Add or remove Datastore access and quotas for a tenant
-    rm                  Delete a tenant
+  {create,vm,update,access,ls,rm}
     create              Create a new tenant
-    ls                  List tenants and the VMs they are applied to
     vm                  Add, removes and lists VMs in a tenant
+    update              Update an existing tenant
+    access              Add or remove Datastore access and quotas for a tenant
+    ls                  List tenants and the VMs they are applied to
+    rm                  Delete a tenant
 
 optional arguments:
   -h, --help            show this help message and exit
 ```
 
 ### Create
+A tenant named "_DEFAULT" should have been created by system automatically before users try to create any tenant.
```

Comment:
A tenant named "_DEFAULT" should have been created by system automatically before users try to create any tenant. => A tenant named "_DEFAULT"  will be created automatically post install.